documentation/player-portal/android/player-config-android.md

```diff
@@ -90,4 +107,65 @@ If the media is DRM-restricted, such as Widevine, the DRM information will be ne
 
 ### PKDrmParams
 
-PKDrmParams represents a single DRM license info object. PKDrmParams contains the licenseUri that will be needed for the play. The PKMediaSource contains a list of "PKDrmParams" items. The player will select the source and the relevant DRM data according to device type, connectivity, supported formats, etc.
\ No newline at end of file
+PKDrmParams represents a single DRM license info object. PKDrmParams contains the licenseUri that will be needed for the play. The PKMediaSource contains a list of "PKDrmParams" items. The player will select the source and the relevant DRM data according to device type, connectivity, supported formats, etc.
+
+
+#### Example for creating MediaEntry with DRM configuration without media provider
+
+```
+   private PKMediaEntry createMediaEntry() {
+        //Create media entry.
+        PKMediaEntry mediaEntry = new PKMediaEntry();
+
+        //Set id for the entry.
+        mediaEntry.setId("testEntry");
+        mediaEntry.setName("testEntryName");
+        mediaEntry.setDuration(881000);
+        //Set media entry type. It could be Live,Vod or Unknown.
+        //In this sample we use Vod.
+        mediaEntry.setMediaType(PKMediaEntry.MediaEntryType.Vod);
+
+        //Create list that contains at least 1 media source.
+        //Each media entry can contain a couple of different media sources.
+        //All of them represent the same content, the difference is in it format.
+        //For example same entry can contain PKMediaSource with dash and another
+        // PKMediaSource can be with hls. The player will decide by itself which source is
+        // preferred for playback.
+        List<PKMediaSource> mediaSources = createMediaSourcesDRM();
+
+        //Set media sources to the entry.
+        mediaEntry.setSources(mediaSources);
+
+        return mediaEntry;
+    }
+    
+    private List<PKMediaSource> createMediaSourcesDRM() {
+
+        //Create new PKMediaSource instance.
+        PKMediaSource mediaSource = new PKMediaSource();
+        mediaSource.setId("1_f93tepsn");
+        //Set the id.
+        mediaSource.setId("drmTestSource");
+
+        String DRM_SOURCE_URL =  "https://cdnapisec.kaltura.com/p/2222401/sp/222240100/playManifest/entryId/1_f93tepsn/protocol/https/format/mpegdash/flavorIds/1_7cgwjy2a,1_xc3jlgr7,1_cn83nztu,1_pgoeohrs/a.mpd";
```

Comment:
We should never use specific IDs for entries, partners, etc.
Please use place holders such as PARTNER_ID, ENTRY_ID.
Same goes for the string in line 151 (/license?custom_data=...)